src/Parser/Core/Modules/Items/ArchimondesHatredReborn.js

```diff
@@ -23,11 +21,7 @@ class ArchimondesHatredReborn extends Module {
   item() {
     return {
       item: ITEMS.ARCHIMONDES_HATRED_REBORN,
-      result: (
-        <dfn data-tip={`The total damaged absorbed by Archimonde's Hatred Reborn was ${this.owner.formatItemAbsorbDone(this.healing)}.`} >
-        {this.owner.formatItemHealingDone(this.healing)}
-        </dfn>
-      ),
+      result:`${this.owner.formatItemHealingDone(this.healing)}`,
```

Comment:
This can be written as `result: this.owner.formatItemHealingDone(this.healing),` since there's no string concatenating happening